.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+  - repo: https://github.com/dnephin/pre-commit-golang
+    rev: v0.5.0
+    hooks:
+    - id: golangci-lint
+    - id: go-unit-tests
```

Comment:
Tests in this repo can take some time, do we really want to run them pre-commit?

Reply:
I'm fine with removing this.
```suggestion
```